rotkehlchen/tests/unit/globaldb/test_migrations.py

```diff
@@ -18,7 +18,7 @@ def test_migration1(globaldb):
     """Test for the 1st globalDB data migration"""
     # Check state before migration
     with globaldb.conn.read_ctx() as cursor:
-        assert globaldb.get_setting_value('version', None) == 6
+        assert globaldb.get_setting_value('version', None) == 7
```

Comment:
You are missing a test for the upgrade in in https://github.com/rotki/rotki/blob/23c611df80e586f390380832a7e58bcb785bc9b9/rotkehlchen/tests/unit/globaldb/test_upgrades.py